hybridse/src/planv2/ast_node_converter.cc

```diff
@@ -1851,8 +1851,15 @@ base::Status ConvertInsertStatement(const zetasql::ASTInsertStatement* root, nod
     }
 
     std::string table_name = "";
-    CHECK_STATUS(AstPathExpressionToString(root->GetTargetPathForNonNested().value(), &table_name));
-    *output = dynamic_cast<node::InsertStmt*>(node_manager->MakeInsertTableNode(table_name, column_list, rows));
+    std::string db_name = "";
+    std::vector<std::string> names;
+    CHECK_STATUS(AstPathExpressionToStringList(root->GetTargetPathForNonNested().value(), names));
+    table_name = names.back();
+    if (names.size() == 2) {
+        db_name = names[0];
+    }
+    *output =
+        dynamic_cast<node::InsertStmt*>(node_manager->MakeInsertTableNode(db_name, table_name, column_list, rows));
```

Comment:
Add unit test in ast_node_converter_test.cc and cases/plan/insert.yaml. Verify the prefix db works